housekeeper/store/filters/archive_filters.py

```diff
@@ -8,25 +8,32 @@
 
 from housekeeper.store.models import Archive
 
-ARCHIVING_ONGOING: BooleanClauseList = and_(Archive.archiving_task_id, Archive.archived_at == None)
-RETRIEVAL_ONGOING: BooleanClauseList = and_(Archive.retrieval_task_id, Archive.retrieved_at == None)
-RETRIEVAL_NOT_ONGOING: BooleanClauseList = not_(
+_ARCHIVING_ONGOING: BooleanClauseList = and_(Archive.archiving_task_id, Archive.archived_at == None)
+_ARCHIVING_COMPLETED: BooleanClauseList = and_(Archive.archiving_task_id, Archive.archived_at)
+_RETRIEVAL_ONGOING: BooleanClauseList = and_(
+    Archive.retrieval_task_id, Archive.retrieved_at == None
+)
+_RETRIEVAL_NOT_ONGOING: BooleanClauseList = not_(
     and_(Archive.retrieval_task_id != None, Archive.retrieved_at == None)
 )
```

Comment:
I was considering changing this to
```suggestion
_RETRIEVAL_NOT_ONGOING: BooleanClauseList = and_(Archive.retrieval_task_id, Archive.retrieved_at)
```
Because we are including entries that have a retrieved_at date but no retrieved job, which would be a malformation of the database. This seems impossible, but at least is more clear in the other way. Anyway, I considered it out of scop